content/en/blog/2023/announcing-major-release.md

```diff
@@ -0,0 +1,106 @@
+
+- [AWS Signer plugin for Notation](https://docs.aws.amazon.com/signer/latest/developerguide/Welcome.html)
+- [Azure Key Vault for Notation](https://learn.microsoft.com/en-us/azure/container-registry/container-registry-tutorial-sign-build-push)
+
+### Integration with admission controller for Kubernetes usage
+
+To enable users to verify and secure image deployment on Kubernetes, the Notary Project maintainers worked with the [Ratify](https://github.com/deislabs/ratify) and [Kyverno](https://kyverno.io/) team respectively to provide solutions for verifying images signed by Notation before deploying them to Kubernetes. Users will have two different options to build a complete end-to-end image integrity workflow for their environments. See the following two guides for details.
+
+- [Sign and verify an image with Notation, Ratify, and OPA Gatekeeper](https://ratify.dev/blog/sign-and-verify-image-with-notation-ratify)
+- [Verify CNCF Notary Project signatures with Kyverno](https://kyverno.io/docs/writing-policies/verify-images/notary/)
+
+![e2e workflow](/e2e-workflow.png)
+
+## Built-in security
+
+The Notary Project also completed fuzzing and security audit in 2023. All vulnerabilities found during the audits are fixed in Notation v1.0.0 (and also present in the prior v1.0.0 RC-7 release). You can find two security audit reports as follows.
+
+- [Notation Security Audit Report 2023](https://github.com/notaryproject/notaryproject/blob/main/security/reports/audit/ADA-notation-security-audit-23.pdf)
+- [Notary Project fuzzing audit report 22-23](https://github.com/notaryproject/notaryproject/tree/main/security/reports/fuzzing/ADA-fuzzing-audit-22-23.pdf)
+
+## What's next
+
+The Notary Project maintainers are considering the following features for the future milestones. If you are interested in the plan or you have new ideas, feel free to let us know in the [Slack channel](https://app.slack.com/client/T08PSQ7BQ/CQUH8U287/) or [GitHub issues](https://github.com/notaryproject/.github/issues).
+
+- Sign and verify any file, even if distributed outside of an OCI registry
+- [GitHub Actions](https://github.com/notaryproject/notation-action) and other CI/CD integration for signing and verification
+- [HashiCorp Vault plugin](https://github.com/notaryproject/notation-hashicorp-vault) (experimental)
+- Plugin lifecycle management
+- Timestamping support
+- Manage trust policy via CLI commands
+
+## Acknowledgements
+
+The Notary Project release team wants to thank the entire Notary Project community for all the activity and engagement that has been vital for helping the project grow and reach this major milestone.
+
+## Try it now
```

Comment:
Shouldn't this be before the Acknowledgements?

Reply:
I think its fine here.

If you do move it, I'd suggest making it the first section before `## What is Notary Project and Notation?` . This approach immediately directs people to try it out.